doc/cephadm/terminology.rst

```diff
@@ -0,0 +1,49 @@
+.. _cephadm-termnilogy:
+
+============
+Termninology
+============
+
+
+`Cephadm` introduces quite a few new terms that need clear demarcations. This article is supposed to go over them and
+explain what they are all about.
+
+
+cephadm (The manager module)
+============================
+
+This is the thing we're talking about when someone is referring to `cephadm`. It is implemented
+as a `mgr module` and is currently the most elaborate orchestrator for ceph.
+Cephadm contains most of the orchestration logic including the `Scheduling` algorithms.
+It's often referred to `mgr/cephadm` to distinguish it from `cephadm` (The tool).
+
+
+cephadm (The daemon tool)
+=========================
+
+This is the tool that operates on a host level and contains logic on how to deploy individual daemons like monitors, managers
+etc. This tool is usually not being used by the end-user but is rather used by the orchestrator module `cephadm`.
+To separate it from it's counterpart, the manager module `cephadm`, it's often referred to `bin/cephadm`.
```

Comment:
This spacing is all a bit odd. I'd recommend wrapping at 80 chars, just as a general rule for reviewing and reading :)